components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -63,11 +63,36 @@ const BillingSubscriptions: React.FC<Props> = () => {
 
     const [showCreditCardBanner, setShowCreditCardBanner] = useState(true);
 
+    const product = useSelector((state: GlobalState) => {
+        if (state.entities.cloud.products && subscription) {
+            return state.entities.cloud.products[subscription?.product_id];
```

Comment:
@nickmisasi Will the subscription have price id or price name so it can be managed by nickname? We need to include this in the Mattermost server models

Reply:
The subscription holds the product ID, and there are already endpoints/state for holding the different possible products. So you should be able to get the product based on the product ID from state and have all of the information you need there.